Framework/include/Utility.h

```diff
@@ -13,6 +13,7 @@ namespace utility
     const std::string color(const std::string& text, const std::string& color);
     std::string split(const std::string& half, const std::string& s, const std::string& h);
     bool compare_p( math::RThetaPhiVector v1, math::RThetaPhiVector v2 );
+    bool compare_pt_TLV( const TLorentzVector v1, const TLorentzVector v2 );
```

Comment:
can you make this a cont TLorentzVector&?

Reply:
Yep!

Reply:
While you are at it can you update the compare_p function to do the same?